provision/rosa-cross-dc/Taskfile.yml

```diff
@@ -0,0 +1,314 @@
+# https://taskfile.dev
+
+version: '3'
+
+output: prefixed
+
+vars:
+  # To compute Infinispan namespace
+  KC_NAMESPACE_PREFIX: '{{default "$(whoami)-" .KC_NAMESPACE_PREFIX}}'
+  KC_ISPN_NAMESPACE: '{{.KC_NAMESPACE_PREFIX}}keycloak'
+  ISPN_DIR: "../infinispan"
+  RDS_DIR: "../aws/rds"
+  KC_DIR: "../openshift"
+  RS_HOT_ROD_PASSWORD:
+    sh: aws secretsmanager get-secret-value --region eu-central-1 --secret-id keycloak-master-password --query SecretString --output text --no-cli-pager
+
+dotenv: [ '.env' ]
+
+includes:
+  common:
+    taskfile: ../common
+    internal: true
+  ispn:
+    taskfile: ../infinispan/Utils.yaml
+    dir: ../infinispan/
+    internal: true
+
+tasks:
+
+  create-aurora:
+    internal: true
+    desc: "Creates a Aurora Database"
+    requires:
+      vars:
+        - AURORA_CLUSTER
+        - AURORA_REGION
+    env:
+      AURORA_CLUSTER: "{{.AURORA_CLUSTER}}"
+      AURORA_REGION: "{{.AURORA_REGION}}"
+    cmds:
+      - echo "Deploying Aurora Database ({{.AURORA_CLUSTER}}) in region {{.AURORA_REGION}}"
+      - ../aws/rds/aurora_create.sh
+
+  create-peering-connection:
+    desc: "Creates Aurora Peering Connection"
+    internal: true
+    silent: false
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - AURORA_CLUSTER
+        - AURORA_REGION
+    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_create_peering_connection.sh
```

Comment:
Can we make use of `env:` here instead of passing as part of the command?

This also applies to many of the other tasks in this file.

Reply:
The semantics are weird when you have some variables in your environment (example, using `export`). You can give it a try:
test.sh
```bash
#!/bin/bash

echo "Message=${MESSAGE}"
```
Taskfile.yaml
```yaml
version: "3"
tasks:
  default:
    - task: print
      vars:
        MESSAGE: "print this"
    - task: print2
      vars:
        MESSAGE: "print this"
  print:
    env:
      MESSAGE: "{{.MESSAGE}}"
    cmds:
      - ./test.sh
  print2:
    cmds:
      - MESSAGE="{{.MESSAGE}}" ./test.sh
```
As you see below, it prefer to uses the value from `export` instead of using the value defined in the yaml file. It caused a lot of headaches so I decided to play safe (and ugly).
```bash
$ export MESSAGE="not this"
$ go-task 
task: [print] ./test.sh
Message=not this
task: [print2] MESSAGE="print this" ./test.sh
Message=print this
```

Reply:
Ok, thanks for explaining :+1: